extension/src/test/e2e/extension.test.ts

```diff
@@ -37,7 +37,7 @@ describe('Experiments Table Webview', function () {
   const webview = new ExperimentsWebview('experiments')
 
   const epochs = 15
-  const headerRows = 3
+  const headerRows = 4
```

Comment:
[N] I would say it's an extra commit row, not a header (not at the top of the table)

Reply:
or specifically, call out "pastCommitsRow"